orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/manifests/CreateBakeManifestTask.java

```diff
@@ -96,11 +100,21 @@ public TaskResult execute(@Nonnull Stage stage) {
 
     String outputArtifactName = expectedArtifacts.get(0).getMatchArtifact().getName();
 
+    Map<String, Object> overrides = (objectMapper.convertValue(context.get("overrides"), new TypeReference<Map<String, Object>>() { }));
+    boolean evaluateOverrideExpressions = (boolean) context.get("evaluateOverrideExpressions");
```

Comment:
This will throw an NPE for stages that don't have this field present; should be a `Boolean` here, or perhaps better could use `stage.mapTo` with a defined model and have Jackson handle the mapping.

Reply:
Here's a recent example of the `.mapTo` thing

Reply:
Would have been helpful if I added the link above: https://github.com/spinnaker/orca/blob/5e88d68f5bfc7588423f2afff8b3f896158269a3/orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/RetryableIgorTask.java#L50

Reply:
Ah thank you, good catch! I will create models for both stage's contexts. I think after that I'll want to do a bit more testing of each stage just to make sure I didn't break anything -- do you know what time the release branch needs to be cut on Monday?

Reply:
Usually not until the afternoon, but I'm sure we can wait if this is almost ready.

Reply:
Perfect! Just pushed an update with the new context models. Let's hold off on merging this until I run through a bunch of test workflows when I'm back at my desktop Monday morning.